src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -39,33 +44,43 @@ const ITEM_TYPES = {
   INPUT: 'input',
   INFO_TEXT: 'infoText',
   TITLE: 'title',
-  CATEGORY: 'category'
+  CATEGORY: 'category',
+  PICKER: 'picker'
 };
 
-// TODO: image and document upload
-//  imageAttributes: {
-//   	title: 'Profil.png',
-//   	cachedAttachment:
-//   		'/Users/ardasenturk/Development/SVA/consul-bb/public/system/images/cached_attachments/user/49/original/3dfe4b17e340624be2f74f822f15e36cadd8d6e1.png'
-//  },
-//  documentsAttributes: [
-// 	  {
-// 		  title: 'sample.pdf',
-// 		  cachedAttachment:
-// 		  	'/Users/ardasenturk/Development/SVA/consul-bb/public/system/documents/cached_attachments/user/49/original/f2245afb48be5f906474ad929aacb7f91f408a23.pdf'
-// 	  }
-//  ],
-
-const showRegistrationFailAlert = () =>
+const imageProps = {
+  onChange: null,
+  allowsEditing: false,
+  aspect: null,
+  quality: 0.2
+};
```

Comment:
why have you decided to create a `const` for `imageProps`. they are only used once and could be passed to `useSelectImage` directly, or am i wrong?
alphabetical order would be nice.
why have you chosen `null` over `undefined`, is there any reason?
why have you chosen `quality: 0.2`? wouldn't `undefined` work here as well to take images as they are?

Reply:
what is used now at the end, when you set it to `undefined`?
and still a question, why decided to create a `const` for `imageProps`. they are only used once and could be passed to `useSelectImage` directly, or am i wrong?

Reply:
You are right. The reason I created `const` is to see their names when sending props to `useSelectImage`. Otherwise I will just have to send data and we will not be able to see which data is 'quality' and which data is 'allowsEditing'. I could not create props as object in `useSelectImage`. It already had the `onChange` prop in it. I just added to it.

Reply:
> to see their names when sending props to `useSelectImage`

i understand. on the other hand, `useSelectImage` is typed and the props are clear. one could easily check whats param 1, 2, 3 or 4. what about:

```
const { imageUri, selectImage } = useSelectImage(
  undefined, // onChange,
  false, // allowsEditing,
  undefined, // aspect,
  undefined // quality
);
```

so the additional variable could be avoided.

have you checked, what is used now at the end, when you set the values to `undefined`?

Reply:
Since there are default values in `imagePicker.ts`, if I enter the `quality` value as undefined, it automatically becomes `quality: 1`. So I can't give undefined value for `quality`. According to my tests, when the highest `quality` value is 0.2, photo uploading takes place. For other props, `null` or `undefined` value works the same.

Reply:
correct, so we must passt a value. and since we do not know better, let's take the 0.2 according to your tests.
what about `aspect`? what can be passed to overwrite the default value with "nothing"?

Reply:
The aspect value can remain `undefined`. To use aspect, the value of `allowsEditing` must be `true`. But we are currently sending it as `false`.

Reply:
8130952b6dccca99da959e64754c14698f3b10bf